[Torchat py] Remove toUnicode method

frozencemetery opened the pull request:
Python includes a method string.unicode() which causes a string (regardless of locale) to be converted to UTF-8.  As the toUnicode() method throws errors on MacOSX (where the default encoding is UTF-8 already and it chokes and dies), this does increase functionality.

prof7bit commented:
I fear this is _might_ produce problems on linux and windows. Did you test your patch on these systems also? If it really works then I would love to merge it because it would remove complexity.

BTW I did not know I had any Mac users at all. Once in a while someone asks me if I could help him getting it running on Mac and I always have to answer that I don't know anything about Mac. Could you please also have a look at this old bug: https://code.google.com/p/torchat/issues/detail?id=98

frozencemetery commented:
My development platform is actually Debian Wheezy, where there are no problems with this change that I can see.  I do not have a windows environment to test this on.

With respect to the old bug you mention, I can write up all of those except for step 4; there are unfortunately other problems beyond the unicode issue that prevent it working on MacOS (at least, for me) which I'm working on.

frozencemetery committed:
Replace toUnicode method with built-in unicode method.

This will enable the code to work on systems where utf8 is the default, such
as Mac OSX.

frozencemetery committed:
Update copyrights.

frozencemetery commented:
I've squashed the commits, but the code is otherwise the same.  This code is now tested working on Linux (Debian Wheezy) and Windows (7).

I'm still working on getting it working on MacOS.  Pull request follows.
